cli/common/utils.go

```diff
@@ -489,6 +497,55 @@ func wrap(i, w int, s string) string {
 	return r
 }
 
+func ParseDependenciesFile(filepath string) ([]PackDependency, error) {
+	content, err := GetFileContent(filepath)
```

Comment:
This `err` variable is re-used by code below. In this case it's better to implicitly declare `var err error` at the top of function.